src/xml/EllipseXML.java

```diff
@@ -0,0 +1,41 @@
+package xml;
+
+import view.PencilState;
+import model.specific_path.Ellipse;
+
+public class EllipseXML extends ShapeXML {
+	
```

Comment:
We already "reserve a position" for the factory in Presentation interface, it's the method `ShapeState getShapeState(Shape shape);`